apple/internal/apple_framework_import.bzl

```diff
@@ -328,7 +433,15 @@ def _apple_dynamic_framework_import_impl(ctx):
         transitive_sets.append(depset(bundling_imports))
     framework_groups = _grouped_framework_files(framework_imports)
     framework_binaries = _all_framework_binaries(framework_groups)
-    dsym_binaries = _all_dsym_binaries(ctx.files.dsym_imports)
+
+    # TODO: Support dSYM import
+    if is_xcframework:
+        dsym_binaries = []
+        dsym_imports = []
```

Comment:
What would this need?

Reply:
Haven't taken a look. I'm going to leave this as a TODO since there doesn't seem to have an absolute need for this right now.